src/EXTRA-DUMP/xdr_compat.h

```diff
@@ -1,100 +1,70 @@
 #ifndef LMP_XDR_COMPAT_H
 #define LMP_XDR_COMPAT_H
 
-#include <climits>
+#include <cstdint>
 #include <cstdio>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 /*
- * This file is needed for systems, that do not provide XDR support
- * in their system libraries. It was written for windows, but will
- * most probably work on other platforms too. better make sure you
- * test that the xtc files produced are ok before using it.
+ * This file contains the definitions for Sun External Data Representation (XDR).
+ * They have been adapted specifically for the use with the LAMMPS xtc dump style
+ * to produce compressed trajectory files in the Gromacs XTC format.
  *
- * It is also needed on BG/L, BG/P and Cray XT3/XT4/XT5 as we don't
- * have XDR support in the lightweight kernel runtimes either.
+ * The XDR sources are avaiable under the BSD 3-clause license for example in
```

Comment:
Found a typo: "avaiable" 

Ditto in xdr_compat.cpp

Reply:
Thanks. Fixed it. Also for "develop"